### ISSUE TITLE
Building simSPI container workflow fails due to size limit.

### ISSUE THREAD

**Issue by fredericpoitevin** [compSPI/simSPI#98]
See https://github.com/compSPI/simSPI/actions/runs/2097918624

I have started work on making the image lighter, but the specific problem with this workflow is that two images are pulled, `simSPI:test` and `simSPI:latest` if the tests pass. Somehow this leads to the error above.

**Comment by ninamiolane** [comment]
@fredericpoitevin do we need to solve this before PIMS?

**Comment by fredericpoitevin** [comment]
I don't think so, it's easy to fix "live" but is a problem outside of hackathon's times.
Having both containers is actually a safety net, but I need to make the whole experience smoother. Another option for the hackathon is to not use the TEM wrapper which allows us to do without the container.
